test/src/helper/spawn.ts

```diff
@@ -160,6 +163,7 @@ export default class CodeChain {
         const { chain, argv, additionalKeysPath, env } = options;
         this._id = CodeChain.idCounter++;
 
+        this._port = 3486 + this.id;
         const { rpcPort = 8081 + this.id } = options;
         this._rpcPort = rpcPort;
```

Comment:
How about not initializing port and rpcPort in the consturctor?
The port numbers initialized here are not used. These values will make a user confuse.

Reply:
I was fixing some minor changes. Sorry for the inconvenience.

Reply:
port and rpcPort cannot be initialized in the constructor, because `getPort` is async so it cannot be called in the constructor.

Reply:
The values initialized there indicate the preference, and they will be used in the most time.
`initialize_port` will change the port into an unbound one only if it's already in use.

Reply:
Although the values are preferenced values, they should not be accessible in the interface. Since they can be accessible by `get port()` and `get rpcPort()` it can be confusing. If you want to set the preferences in the constructor function, please create preferredRPCPort and preferredPort fields.

Reply:
I misread what you commented. sorry. You are right. I'll remove it.